common/src/main/java/com/wynntils/wynn/objects/SpellType.java

```diff
@@ -14,7 +14,7 @@ public enum SpellType {
     ARROW_SHIELD(ClassType.Archer, 4, "Arrow Shield", 8, 1),
 
     SPIN_ATTACK(ClassType.Assassin, 1, "Spin Attack", 6, 0),
-    VANISH(ClassType.Assassin, 2, "Vanish", 2, 0),
+    VANISH(ClassType.Assassin, 2, "Dash", 2, 0),
```

Comment:
Has the spell changed name in 2.0? Maybe we should update the name of the enum as well?

Reply:
It did, and I agree